client/src/app/components/stardust/section/referenced-blocks/ReferencedBlocksSectionRow.tsx

```diff
@@ -64,9 +60,7 @@ const ReferencedBlocksSectionRow: React.FC<Props> = ({ blockId, isTable }) => {
 
             setIsLoading(false);
         });
-
-        return unmount;
-    }, [blockId]);
+    }, [isMounted, blockId]);
```

Comment:
Why the `isMounted` as dependency ?  Won't this run the useEffect every time "isMounted" changes ? I don't see the point.